src/api/app/controllers/webui/staging_workflows_controller.rb

```diff
@@ -1,5 +1,9 @@
 class Webui::StagingWorkflowsController < Webui::WebuiController
```

Comment:
I am confused... isn't this class created in https://github.com/openSUSE/open-build-service/pull/6073 which is not merged yet? :confused: It is a :ghost: